### PR TITLE
Some changes to README based on discussion with L&D team

### PR DESCRIPTION
**Pull request by amsaha**

### CODE REVIEW

File: tf-mnist/README.md [hunk 1]
```diff
@@ -323,6 +326,17 @@ variables in variables.bash and run the following script:
 After running this script, you will get the IP adress of your web application.
 Open browser and see app at http://IP_ADRESS:NodePort
 
+# Completion Criteria
+
+This exercise can be considered complete once the application has been verified
+by [using a python client](#using-a-local-python-client) AND
+by [using a web browser](#using-a-web-application). In the below image,
```
Comment (ramdootp) [inline tf-mnist/README.md]: https://github.com/CiscoAI/KFLab/blob/5d93c558a9cf257eaa57bf2c95c1c8b9c6bac212/tf-mnist/README.md#port-forwarding - There are multiple ways in web application. Port forwarding way is a good completion criteria, else there might be confusion :-)

Comment (amsaha) [reply tf-mnist/README.md]: So, we can clarify that either "Port Forwarding" (which is what we do now) or NodePort (we use to do that earlier) should be the completion criteria and not the Load Balancer, right? 
BTW, currently, are we even doing NodePort way? Is the information at https://github.com/CiscoAI/KFLab/blob/master/tf-mnist/README.md#nodeport relevant/correct?

Comment (ramdootp) [reply tf-mnist/README.md]: There was an issue with NodePort, hence commented out temporraily.

**Pull request merged**